### PR TITLE
adding annotated metagenome assembly to Narrative Viewers

### PR DESCRIPTION
**Pull request by slebras**

### CODE REVIEW

File: ui/narrative/methods/view_annotated_metagenome_assembly/display.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+#
+# Configure the display and description of the parameters
+#
+parameters :
+    param0 :
+        ui-name : |
+            Annotated Metagenome Assembly
+        short-hint : |
+            select the genome you want to view [5.1]
```
Comment (briehl) [inline ui/narrative/methods/view_annotated_metagenome_assembly/display.yaml]: the [number] stuff isn't necessary.

Comment (briehl) [reply ui/narrative/methods/view_annotated_metagenome_assembly/display.yaml]: in all the fields

**Review by briehl — CHANGES_REQUESTED** [review]
just a little cleanup, but it's fine.

File: ui/narrative/methods/view_annotated_metagenome_assembly/display.yaml (hunk 1 above)
Comment (slebras) [reply ui/narrative/methods/view_annotated_metagenome_assembly/display.yaml]: donezo

**Pull request merged**